Bug Reporting/ARKBugReporter.h

```diff
@@ -26,9 +26,12 @@ NS_ASSUME_NONNULL_BEGIN
 
 @protocol ARKBugReporter <NSObject>
 
-/// Called when the user has triggered the creation of a bug report.
+/// Called when the user has triggered the creation of a bug report. Logs a screenshot by default.
 - (void)composeBugReport;
 
+/// Called when the user has triggered the creation of a bug report, optionally, logs a screenshot.
+- (void)composeBugReportWithScreenshotLog:(BOOL)logScreenshot;
```

Comment:
Not to open a can of worms, but `composeBugReportShouldLogScreenshot:` sounds like the return type should be a BOOL. How about `composeBugReportAndLogScreenshot:`?

I also like `composeBugReport` and `composeBugReportWithoutScreenshot`